draft-ietf-mls-protocol.md

```diff
@@ -680,16 +680,22 @@ they receive the private keys for nodes, as described in
 
 ## Ratchet Tree Commits
 
-Nodes in a tree are always updated along the direct path from a
-leaf to the root.  The generator of the Commit chooses a random
-secret value "path_secret[0]", and generates a sequence of "path
-secrets", one for each node from the leaf to the root.  That is,
-path_secret[0] is used for the leaf, path_secret[1] for its parent,
-and so on.  At each step, the path secret is used to derive a new
-secret value for the corresponding node, from which the node's key
-pair is derived.
+When performing a Commit, the leaf ClientInitKey of the commiter and
+its direct path to the root are updated with new secret values.
+
+The generator of the Commit starts by using the HPKE secret key
+"leaf_hpke_secret_key" associated to the "init_key" of its new leaf
+CIK (See {{initialization-keys}}) to compute the "path_secret[0]"
```

Comment:
- we've never abbreviated this before, CIK -> ClientInitKey
- lowercase, See -> see
- remove "the" in front of "path_secret[0]"

Reply:
Thanks !